wurst/data/HashMap.wurst

```diff
@@ -29,6 +33,17 @@ public class HashMap<K,V> extends Table
 		remove(key)
 		return result
 
+	/** Remove all data from this map */
+	override function flush()
+		size = 0
+		super.flush()
+
+	function size()returns int
+		return size
+
+	ondestroy
```

Comment:
seems pointless? Shouldn't call anything on a destroyed instance.

Reply:
Accessing values of destroyed instances is invalid

Reply:
i already maked a new commit